pkg/pb/metainfo.proto

```diff
@@ -42,12 +42,13 @@ service Metainfo {
 message Bucket {
```

Comment:
**Rational for code Changes:**
The changes in this protobuf file is due to the addition of partner_id field